conference/DS/ica3pp.yml

```diff
@@ -0,0 +1,26 @@
+- title: ICA3PP
+  description: International Conference on Algorithms and Architectures for Parallel Processing
+  sub: DS
+  rank: A
```

Comment:
It should be CCF C?

Reply:
Yes, source: https://www.ccf.org.cn/Academic_Evaluation/ARCH_DCP_SS